docs/make.jl

```diff
@@ -1,4 +1,49 @@
-using Documenter, GAP
+using Documenter, GAP, Markdown
+
+abstract type ExternalReference <: Documenter.Builder.DocumentPipeline end
```

Comment:
Some comments in this file that give an idea what's going on here would be really nice. Perhaps also a link to https://github.com/JuliaDocs/Documenter.jl/issues/1343 ?